test/runtests.cmd

```diff
@@ -323,15 +323,10 @@ goto :main
     set _DIRS=-all
   )
 
-  set _BuildArchMapped=%_BuildArch%
   set _BuildTypeMapped=%_BuildType%
 
-  :: Map new build arch and type names to old names until rl test tags are
-  :: updated to the new names
-  if "%_BuildArchMapped%" == "x64" set _BuildArchMapped=amd64
-  if "%_BuildTypeMapped%" == "debug" set _BuildTypeMapped=chk
-  if "%_BuildTypeMapped%" == "test" set _BuildTypeMapped=fre
-  if "%_BuildTypeMapped%" == "codecoverage" set _BuildTypeMapped=fre
+  :: codecoverage need to mapped to test config

```

Comment:
I was hoping we could remove both `Mapped` environment variables.  `codecoverage` puts an annoying thorn in that.  Maybe this can be a separate option?